### ISSUE TITLE
HyPhy 2.5.58 

### ISSUE THREAD

**Issue by anivanoff** [ubccr/software-layer#142]
Please help me install this latest version of HyPhy, source code available here: https://github.com/veg/hyphy/releases/tag/2.5.58

**Comment by anivanoff** [comment]
other potentially useful links: 
https://anaconda.org/bioconda/hyphy/files 
https://bioconda.github.io/recipes/hyphy/README.html

**Comment by dsajdak** [comment]
See ticket 34649 for test script

**Comment by tonykew** [comment]
Test build completed - more info to follow

**Comment by tonykew** [comment]
Built the latest version: 2.5.59 - more info to follow

**Comment by dsajdak** [comment]
Hi @anivanoff hyphy has been published into the `ccrsoft/2023.01` software release:

```
vortex:~$ module spider hyphy

----------------------------------------------------------------------------------------------------------------
  hyphy: hyphy/2.5.59
----------------------------------------------------------------------------------------------------------------
    Description:
      HyPhy (Hypothesis Testing using Phylogenies) is an open-source software package for the analysis of
      genetic sequences (in particular the inference of natural selection) using techniques in phylogenetics,
      molecular evolution, and machine learning

    Properties:
      Bioinformatic libraries/apps

    You will need to load all module(s) on any one of the lines below before the "hyphy/2.5.59" module is available to load.

      gcc/11.2.0  openmpi/4.1.1

    Help:

      Description
      ===========
      HyPhy (Hypothesis Testing using Phylogenies) is an open-source software package
       for the analysis of genetic sequences (in particular the inference of natural selection)
       using techniques in phylogenetics, molecular evolution, and machine learning


      More information
      ================
       - Homepage: https://veg.github.io/hyphy-site/

vortex:~$ module load gcc/11.2.0  openmpi/4.1.1 hyphy/2.5.59
```
